src/k8s-extension/azext_k8s_extension/_client_factory.py

```diff
@@ -5,17 +5,33 @@
 
 from azure.cli.core.commands.client_factory import get_mgmt_service_client
 from azure.cli.core.profiles import ResourceType
+from . import consts
 
-
-def cf_k8s_extension(cli_ctx, *_):
+def cf_k8s_extension(cli_ctx, **kwargs):
```

Comment:
> Running flake8 on extensions...
> /home/vsts/work/1/s/src/k8s-extension/azext_k8s_extension/_client_factory.py:10:1: E302 expected 2 blank lines, found 1
1     E302 expected 2 blank lines, found 1
Please fix the CI issue

```suggestion

def cf_k8s_extension(cli_ctx, **kwargs):
```